src/core/border_router/routing_manager.cpp

```diff
@@ -738,13 +738,13 @@ void RoutingManager::EvaluateRoutingPolicy(void)
     if (newOmrPrefixes.IsEmpty())
     {
         // This is the very exceptional case and happens only when we failed to publish
-        // our local OMR prefix to the Thread network. We schedule the Router Advertisement
+        // our local OMR prefix to the Thread network. We schedule the routing policy
         // timer to re-evaluate our routing policy in the future.
 
-        LogWarn("No OMR prefix advertised! Start Router Advertisement timer for future evaluation");
+        LogWarn("No OMR prefix advertised! Start Routing Policy timer for future evaluation");
     }
 
-    // 2. Schedule Router Advertisement timer with random interval.
+    // 2. Schedule routing policy timer with random interval.
```

Comment:
I think the purpose of the code is to schedule the next RA. So, maybe we should state that the purpose is to send next RA.

```suggestion
    // 2. Schedule routing policy timer with random interval for the next Router Advertisement.
```

Reply:
Updated